ruby/red-arrow/ext/arrow/arrow.cc

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Could you use `arrow.cpp` instead of `arrow.cc`?
Because we use `.cpp` in Arrow GLib and we use `.hpp` for header file.